addons/website_sale_stock/tests/test_website_sale_stock_product_warehouse.py

```diff
@@ -74,3 +74,25 @@ def test_get_combination_info(self):
 
             # Check available quantity of product is according to warehouse
             self.assertEqual(combination_info['virtual_available'], qty_b, "%s units of Product B should be available in warehouse %s" % (qty_b, wh))
+
+    def test_update_cart_with_multi_warehouses(self):
```

Comment:
Can you please number it? eg `test_01_xx` and 02 so we start on a clean basis ? thanks